scripts/requirements.txt

```diff
@@ -9,6 +9,7 @@ ply==3.10
 hub==2.0
 gitlint
 pyelftools==0.24
+pyocd
```

Comment:
should this be:

    pyocd>=0.11.0

Reply:
I don't think so. Not including the version specifier means install the latest version, which as of today is 0.11.1.